frontend/src/citizen-frontend/messages/MessagesPage.tsx

```diff
@@ -121,6 +133,7 @@ export default React.memo(function MessagesPage() {
                     closeThread={() => selectThread(undefined)}
                     thread={selectedThread}
                     onThreadDeleted={() => onSelectedThreadDeleted()}
+                    setInitFocusFunc={(f) => (initThreadFocusRef.current = f)}
```

Comment:
Täältä ongitaan titlen referenssi
```suggestion
                    ref={threadTitleRef}
```